dockerfiles/jupyter/Dockerfile

```diff
@@ -41,7 +42,8 @@ ENV MLRUN_ARTIFACT_PATH=$HOME/data \
     MLRUN_HTTPDB__DATA_VOLUME=$HOME/data \
     MLRUN_HTTPDB__DSN='sqlite:////home/jovyan/data/mlrun.db?check_same_thread=false' \
     MLRUN_HTTPDB__LOGS_PATH=$HOME/data/logs \
-    MLRUN_PVC_MOUNT=nfsvol:/home/jovyan/data
+    MLRUN_PVC_MOUNT=nfsvol:/home/jovyan/data \
+    MLRUN_SET_ENV_FILE=$HOME/env
```

Comment:
I suggest to rename to `MLRUN_CONFIG_FILE` or `MLRUN_ENV_FILE` if you feel like separating between the config and the env